### DISCUSSION TITLE
2023-08-02 Weekly Meeting 118 Notes/Agenda

### DISCUSSION THREAD

**Discussion by madjin** [omigroup/omigroup#409] [General]
Weekly Meeting 118 is happening on Wednesday, August 2, 2023 from 9:00 AM PST (16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda
notes from last week: https://github.com/omigroup/omigroup/discussions/408
champions four weeks ago: https://github.com/omigroup/omigroup/discussions/404
new readme: https://github.com/omigroup/
MSF POG status
we are approved and finished with the application.
now its time to nominate to the board (I believe, can we confirm?)
Open Collective Update

- Funding ideas - how can we help people fund their ideas? see [OMI DIscord Experiments Funding thread](https://discord.com/channels/770382203782692945/1128733053716549722) for discussion
- Metaverse Festival October 2024 see [HackMD draft information for page on OMI website to start spreading the word](https://hackmd.io/@indiebio/metaverse-float-parade)

**UPDATES**

comments below

**Comment by indiebio** [comment]
@jin shares progress on the festival idea, some vehicles, booths are available on https://sketchfab.com/m3org/models

**Comment by indiebio** [comment]
Media: have working meetings that can be livestreamed and recorded, aka creating media, while we work on e.g. creating the content for a press kit for the festival idea (separate meeting to the next media meetup which will be about access and accounts.
TODO: Schedule these at a reasonable frequency.

**Comment by madjin** [comment]
- vket
  - vket summer is over, but we went on several field trips / took lots of pics, MSF avatar / character group learneed about it last night through a presentation by someone from Hikky
- MSF delegates
  - MSF update: some interest for a virtual expo from Neil, noted presence and expression limitations of virtual world (I think vrchat can fix this). The interoperable avatar/chars group had tour of vket from someone in Hikky last night. I think if we just keep building booths and parade vehicles others will eventually catch on, and we can have sponsorship packages for companies that wanna outsource creation.
- Media group
  - Meeting coming up soon and backlog meeting coming up soon as well.
  - https://www.twitch.tv/anarchyarcade  Example livestreamed talk show via discord / green screen background removal
![image](https://github.com/omigroup/omigroup/assets/32600939/77278e07-a6f0-4ae5-9513-843852b1cf62)
![image](https://github.com/omigroup/omigroup/assets/32600939/60fa3b0b-5dfb-4ef7-a9ae-58fcaf72f2f2)

- Avatar interop group
  - went over wearables interop challenges and ideas with a member of the cyberbrokers team https://snap.cyberbrokers.com/, recorded it via zoom / fireflies AI - pretty useful software we should try uploading past OMI meeting recordings to it https://fireflies.ai/
  - Can upload transcription / notes later to Github or something
- GLTF group
  - Meeting with Khronos group last week, some highlights:
    - Some context before, didn't have much idea what Khronos group wanted yet since wasn't much async discussion prior. For meeting talked about scope of the spec, groups wants it to be big and flexible, as result will spend more time working on OMI specs to bring closer to MSFT specs and implement things Khronos group wants for sure, and converge into a KHR physics spec. Still have portability concerns, proposed filtering cannot be implemented in Godot atm (shape / body stuff), but it's just a caveat we gotta deal with atm.
    - Will keep making gradual changes over the month

**Comment by indiebio** [comment]
Please find the recordings of the meetings, and if possible add each meeting's recording link to the meeting minutes.
